app/src/androidTest/java/com/github/se/signify/ui/screens/home/HomeScreenTest.kt

```diff
@@ -220,4 +221,21 @@ class HomeScreenTest {
       composeTestRule.onNodeWithTag("ScrollToTopButton").performClick()
     }
   }
+
+  @Test
+  fun dictionaryIsDisplayedUsingPagersScrolling() {
+    composeTestRule.setContent { HomeScreen(navigationActions) }
+
+    ('A'..'Z').forEachIndexed { _, letter ->
```

Comment:
Suggestion: Consider testing edge cases explicitly, such as attempting to swipe beyond 'A' or 'Z', to ensure the system handles such actions.